src/redis.ts

```diff
@@ -0,0 +1,1256 @@
+  }
+
+  getrange(key: string, start: number, end: number, collection: string) {
+    const command: string = "GETRANGE";
+    return this._commandParser(command, collection, key, start, end);
+  }
+
+  getset(key: string, value: string, collection: string) {
+    const command: string = "GETSET";
+    return this._commandParser(command, collection, key, value);
+  }
+
+  incr(key: string, collection: string) {
+    const command: string = "INCR";
+    return this._commandParser(command, collection, key);
+  }
+
+  incrby(key: string, increment: number, collection: string) {
+    const command: string = "INCRBY";
+    return this._commandParser(command, collection, key, increment);
+  }
+
+  incrbyfloat(key: string, increment: number, collection: string) {
+    const command: string = "INCRBYFLOAT";
+    return this._commandParser(command, collection, key, increment);
+  }
+
+  mget(keys: string[], collection: string) {
+    const command: string = "MGET";
+    return this._commandParser(command, collection, ...keys);
+  }
+
+  mset(data: object, collection: string) {
+    const dataArray: string[] = [];
+    for (const [key, value] of Object.entries(data)) {
+      dataArray.push(key);
+      dataArray.push(value);
+    }
+    const command: string = "MSET";
+    return this._commandParser(command, collection, ...dataArray);
+  }
+
+  psetex(key: string, milliseconds: number, value: number, collection: string) {
+    const command: string = "PSETEX";
+    return this._commandParser(command, collection, key, milliseconds, value);
+  }
+
+  setbit(key: string, offset: number, value: number, collection: string) {
+    const command: string = "SETBIT";
+    return this._commandParser(command, collection, key, offset, value);
+  }
+
+  msetnx(data: object, collection: string) {
+    const dataArray: string[] = [];
+    for (const [key, value] of Object.entries(data)) {
+      dataArray.push(key);
+      dataArray.push(value);
+    }
+    const command: string = "MSETNX";
+    return this._commandParser(command, collection, ...dataArray);
+  }
+
+  setex(key: string, seconds: number, value: string, collection: string) {
+    const command: string = "SETEX";
+    return this._commandParser(command, collection, key, seconds, value);
+  }
+
+  setnx(key: string, value: string, collection: string) {
+    const command: string = "SETNX";
+    return this._commandParser(command, collection, key, value);
+  }
+
+  setrange(key: string, offset: number, value: string, collection: string) {
+    const command: string = "SETRANGE";
+    return this._commandParser(command, collection, key, offset, value);
+  }
+
+  strlen(key: string, collection: string) {
+    const command: string = "STRLEN";
+    return this._commandParser(command, collection, key);
+  }
+
+  bitcount(
+    key: string,
+    collection: string,
+    start?: number | undefined,
+    end?: number | undefined,
+    dataFormat?: string | undefined
```

Comment:
the optional property will be considered undefined. no need to pass undefined specifically.

Reply:
Applicable to all similar instances.

Reply:
👍